ui/ui-frontend/projects/identity/src/app/customer/owner-form/owner-form.component.html

```diff
@@ -20,27 +20,15 @@
   </vitamui-common-input>
 
   <div class="d-flex">
-    <vitamui-common-input
-      class="col-6 pl-0"
-      formControlName="name"
-      maxlength="100"
-      required
-      [placeholder]="'CUSTOMER.OWNER.NAME' | translate"
-    >
+    <vitamui-common-input class="col-6 pl-0" formControlName="name" required [placeholder]="'CUSTOMER.OWNER.NAME' | translate">
       <vitamui-common-input-error *ngIf="form.get('name')?.touched && !!form.get('name')?.errors?.required"
         >{{ 'COMMON.ERROR.REQUIRED' | translate }}
       </vitamui-common-input-error>
       <vitamui-common-input-error *ngIf="!!form?.get('name')?.errors?.maxlength"
         >{{ 'COMMON.ERROR.STRING_MAX_LENGTH' | translate: { max: ownerNameMaxLength } }}
```

Comment:
```suggestion
        >{{ 'COMMON.ERROR.STRING_MAX_LENGTH' | translate: { max: form!.get('name')!.errors!.maxlength!.requiredLength } }}
```
idem pour les autre occurrences.